Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:13
```

Comment:
Don't use unstable node (all uneven numbers are inherently not stable and will _never_ reach LTS). Instead use lts, or use debian and install node.